src/stratis_cli/_actions/_logical.py

```diff
@@ -79,8 +79,13 @@ def list_volumes(namespace):
            managed_objects,
            props={'Pool': pool_object_path}
         )
+        fs_names = []
```

Comment:
This would be nicer if more functional, like:

```
fs_names = \
   [name for name in map(lambda p: MOFilesystem(p[1]).Name(), matching_filesystems)]
for fs in sorted(fs_name):
    print(fs)
```

Also, I'm not sure if key should really be lowercase version of str. Might as well use the same key as their actual value.